### PR TITLE
Better header override and CSS clean

### PR DESCRIPTION
**Pull request by nboisteault**

### CODE REVIEW

File: lizmap/www/assets/css/main.css [hunk 1]
```diff
@@ -35,7 +20,8 @@ a:hover {
   z-index: 1030;
   margin-bottom: 0px;
   overflow: visible;
-  background : url('img/header-background-medium.jpg') repeat fixed #2B2B2B;
+  background-color: #2B2B2B;
+  background-image : url(img/header-background-medium.jpg);
```
Comment (mdouchin) [inline lizmap/www/assets/css/main.css:24]: I would add a background-repeat and the other config to set the "fixed" property, to be sure on big screens, there is no blank (the image must be repeated)

File: lizmap/www/themes/default/css/main.css [hunk 2]
```diff
@@ -2,21 +2,9 @@ body {
...
-a:hover {
-  color:white\9;
-  text-decoration:none\9;
-}
-/* END for ie 8*/
-
 #header {
-  background : url(img/header-background-medium.jpg) repeat fixed #2B2B2B;
+  background-color: #2B2B2B;
+  background-image : url(img/header-background-medium.jpg);
```
Comment (mdouchin) [inline lizmap/www/themes/default/css/main.css:7]: same here ?

**Review by mdouchin — CHANGES_REQUESTED** [review]

File: lizmap/www/assets/css/main.css (hunk 1 above)
Comment (nboisteault) [reply lizmap/www/assets/css/main.css:24]: If I set `background-attachment : fixed`, the black color goes above image. Default is `background-attachment : scroll`. As there is no scroll in the element I think we should let default.

Comment (nboisteault) [reply lizmap/www/assets/css/main.css:24]: I've added `background-repeat: repeat;`.

**Review by mdouchin — APPROVED** [review]

**Pull request merged**